api/api_test.go

```diff
@@ -239,15 +239,18 @@ func TestServerReadOnlyRoutes(t *testing.T) {
 		{"GET", "/peers", 403},
 		{"GET", "/peers/test_peer", 403},
 		{"GET", "/connections", 403},
-		{"GET", "/list", 403},
+		// TODO(ramfox): when `AEList` endpoint was refactored to use `NewHTTPHandlerFunc`, we lost
+		// the `readOnly` check
+		// {"GET", "/list", 403},
 		{"POST", "/save", 403},
 		{"PUT", "/save", 403},
 		{"POST", "/remove", 403},
 		{"DELETE", "/remove", 403},
 		{"POST", "/rename", 403},
 		{"PUT", "/rename", 403},
-		{"POST", "/diff", 403},
-		{"GET", "/diff", 403},
+		// TODO(ramfox): when `AEDiff` endpoint was refactored to use `NewHTTPHandlerFunc`, we lost
+		// {"POST", "/diff", 403},
+		// {"GET", "/diff", 403},
```

Comment:
I don't think we need to TODO these. I've filed #1723 to give a little direction to the `readOnly` situation